src/styles/_githubAssessments.scss

```diff
@@ -0,0 +1,192 @@
+.MissionEditor {
+  height: 100%;
+  display: flex;
+  background-color: $cadet-color-1;
+  flex-direction: column;
+  flex: 1 1 100%;
+
+  .Multi-line {
+    white-space: 'pre-line';
+    overflow-wrap: 'break-word';
+  }
+}
```

Comment:
i think this css is unused, or named wrongly

Reply:
Style deleted:
https://github.com/chekjun/cadet-frontend/blob/master/src/styles/_githubAssessments.scss